jmx_prometheus_httpserver/pom.xml

```diff
@@ -30,6 +30,11 @@
       <artifactId>jetty-servlet</artifactId>
       <version>8.1.7.v20120910</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlets</artifactId>
+      <version>8.1.7.v20120910</version>
```

Comment:
While you're here, would you mind upgrading us to something recent to solve #94?